velox/common/caching/SsdFile.cpp

```diff
@@ -1021,10 +1007,16 @@ void SsdFile::readCheckpoint(std::ifstream& state) {
     idMap[id] = StringIdLease(fileIds(), id, name);
   }
 
-  const auto logSize = ::lseek(evictLogFd_, 0, SEEK_END);
+  const auto logPath = getEvictLogFilePath();
+  const auto evictLogReadFile = fs_->openFileForRead(logPath);
+  const auto logSize = evictLogReadFile->size();
   std::vector<uint32_t> evicted(logSize / sizeof(uint32_t));
-  const auto rc = ::pread(evictLogFd_, evicted.data(), logSize, 0);
-  VELOX_CHECK_EQ(logSize, rc, "Failed to read eviction log");
+  try {
+    evictLogReadFile->pread(0, logSize, evicted.data());
+  } catch (const std::exception& e) {
+    ++stats_.readCheckpointErrors;
```

Comment:
This seems to be a separate error like readEvictionLogErrors. Could separate this in followup. Thanks!